frontend/src/pages/home/index.tsx

```diff
@@ -38,7 +52,18 @@ const HomePage: React.FC = () => {
           </Grid>
         </Grid>
         <Grid item container columns={18} spacing={2}>
-          <Grid item md={11}></Grid>
+          <Grid item md={11}>
+            <WeeklyActionItemsPanel
+              height={295}
+              data={actionItemsQuery.status == 'success' ? actionItemsQuery.data.actionItems : []}
+              itemsID={
```

Comment:
What is this prop?

Reply:
Because the action items themself contained in weeklyactionitems so we need to send the weeklyactionitems _id so we know who to update, thus avoiding itrating over every weekltactionitems to search the actionitem to update

Reply:
I added a comment regarding that